tests/task_router/test_workflow_config.py

```diff
@@ -78,6 +78,108 @@ def test_from_json2(self):
+
+            # convert back to json; should marshal as friendly_name
+            config_json = config.to_json()
+            expected_config_data = {
+                                      "task_routing":
+                                      {
+                                          "default_filter":
+                                          {
+                                              "queue": "WQ05f810d2d130344fd56e3c91ece2e594"
+                                          },
+                                          "filters": [
+                                              {
+                                                  "expression": "type == \"sales\"",
+                                                  "friendly_name": "Sales",
+                                                  "targets": [
+                                                      {
+                                                          "expression": "task.language IN worker.languages",
+                                                          "queue": "WQec62de0e1148b8477f2e24579779c8b1"
+                                                      }
+                                                  ]
+                                              },
+                                              {
+                                                  "expression": "type == \"marketing\"",
+                                                  "friendly_name": "Marketing",
+                                                  "targets": [
+                                                      {
+                                                          "expression": "task.language IN worker.languages",
+                                                          "queue": "WQ2acd4c1a41ffadce5d1bac9e1ce2fa9f"
+                                                      }
+                                                  ]
+                                              },
+                                              {
+                                                  "expression": "type == \"support\"",
+                                                  "friendly_name": "Support",
+                                                  "targets": [
+                                                      {
+                                                          "expression": "task.language IN worker.languages",
+                                                          "queue": "WQe5eb317eb23500ade45087ea6522896c"
+                                                      }
+                                                  ]
+                                              }
+                                          ]
+                                      }
+                                   }
+
+            expected_config_json = json.dumps(expected_config_data,
+                          default=lambda o: o.__dict__,
```

Comment:
is `default=lambda o: o.__dict__,` necessary when `expected_config_data` is a dict?